Gruntfile.js

```diff
@@ -1,10 +1,24 @@
 module.exports = function (grunt) {
+	var staticExampleFiles = [ 'src/examples/**', '!src/examples/**/*.js' ];
```

Comment:
The `examples` folder would contain a sub-folder for each widget, each with an HTML page. These pages would just create the respective widget, not exhaustively test them. This helps for widget creation and adding new features or fixing bugs, as it allows you to see if things work immediately when running `grunt dev`. If we put these small test pages in `dojo/examples`, developers would have to manually link `dojo/widgets` locally to test their latest changes. I also see `dojo/examples` as more polished and thorough widget examples.